coalib/coala_json.py

```diff
@@ -28,12 +31,38 @@ def main():
     args = arg_parser.parse_args()
 
     log_printer = None if args.text_logs else ListLogPrinter()
-    results, exitcode, _ = run_coala(log_printer=log_printer, autoapply=False)
+    JSONEncoder = create_json_encoder(use_relpath=args.relpath)
+
+    if args.show_bears:
+        try:
+            local_bears, global_bears = get_filtered_bears(
+                args.filter_by_language, log_printer)
 
-    retval = {"results": results}
+            bears = inverse_dicts(local_bears, global_bears)
+            results = []
+            for bear, sections in sorted(bears.items(),
+                                         key=lambda bear_tuple:
+                                         bear_tuple[0].name):
+                metadata = bear.get_metadata()
+                results.append({"Name": bear.name,
+                                "Description": metadata.desc,
+                                "Supported languages": bear.LANGUAGES,
```

Comment:
Possibly add a try/catch and catch TypeErrors. And print which property is causing this ?
Also, `bear`s have sets right ? For LANGUAGE, etc etc ... So, I don't see how everything can be a string.
`set()` is not JSON serializable:
```
In [2]: a = {"a": set([1,2])}
In [3]: a
Out[3]: {'a': {1, 2}}
In [4]: import json
In [5]: json.dumps(a)
...
TypeError: set([1, 2]) is not JSON serializable
```